.github/workflows/deploy.yml

```
@@ -53,7 +55,9 @@ jobs:
         run: |
           cd /home/selab2/Dwengo-4/
           git checkout develop
```

Main zou moeten gedployed worden
```suggestion
          git checkout main
```

---

Moet dit hier nog staan want is het niet beter om `actions/checkout` te gebruiken?